Cargo.toml

```diff
@@ -32,7 +33,7 @@ no-dev-version = true
 tag-name = "{{version}}"
 
 [package.metadata.docs.rs]
-features = ["serde"]
+features = ["serde", "borsh"]
 
 [package.metadata.playground]
-features = ["serde"]
+features = ["serde", "borsh"]
```

Comment:
Borsh isn't currently in the playground, so I'm not sure it would be courteous of us to artificially bump it in, or all that useful since other crates which provide borsh as an optional feature don't do so in the context of the playground.